Subproject2/1. SOVA/Controllers/CommentsController.cs

```diff
@@ -15,7 +15,7 @@ namespace _1._SOVA.Controllers
     public class CommentsController : ControllerBase
     {
         private readonly ICommentRepository _commentRepository;
-        private IMapper _mapper;
+        private readonly IMapper _mapper;
```

Comment:
why do we need readonly here?

Reply:
Resharper changed that to readonly. It doesn't really affect the functionality but I'll change it back.